packages/compass-e2e-tests/helpers/commands/export-to-language.ts

```diff
@@ -73,5 +73,5 @@ export async function exportToLanguage(
   await exportModal.waitForDisplayed({ reverse: true });
 
   // normalize copied text so that it's the same for all platforms
-  return text.replace(/\r\n/g, '\n');
+  return text.replace(/\r\n/g, '\n').replace(/\n+/g, '\n');
```

Comment:
For whatever reason now on some operating systems there's an extra linebreak sometimes, but only from the perspective of webdriver. This just tries to normalise the text we get from export-to-language some more to make assertions easier.